src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -271,8 +266,8 @@ ov::AnyMap get_default_prefill_config(const std::shared_ptr<ov::Model>& model,
         return { };
     }
     auto config = get_default_common_config(model);
-    if (get_option<std::string>(config, "NPUW_FUNCALL_FOR_ALL").value_or("NO") == "YES") {
-        config.emplace("NPUW_PARALLEL_COMPILE", "YES");
+    if (is_cw_compressed(model)) {
+        config.emplace("NPUW_DQ", "YES");
```

Comment:
If we don't enable DQ (see `else`), I'd also recommend to disable PMM (`PMM : NO`)

Reply:
Done